src/pages/login/Login.tsx

```diff
@@ -111,7 +113,7 @@ const LoginButton = ({
       rightIcon={IconNames.LOG_IN}
       onClick={useCallback(() => handleClick(id), [handleClick, id])}
     >
-      {`Log in with ${name}`}
+      <Translation ns={'login'}>{t => t('Log in with', { name: name })}</Translation>
```

Comment:
This string can use a normal attribute:

```suggestion
      <Translation ns="login">{t => t('Log in with', { name: name })}</Translation>
```